src/test/java/no/nav/veilarboppfolging/service/IservServiceIntegrationTest.java

```diff
@@ -69,7 +71,7 @@ public void behandleEndretBruker_skalLagreNyIservBruker() {
         IservMapper iservMapper = utmeldingRepository.eksisterendeIservBruker(veilarbArenaOppfolging);
         assertThat(iservMapper).isNotNull();
         assertThat(iservMapper.getAktor_Id()).isEqualTo(AKTORID);
-        assertThat(iservMapper.getIservSiden()).isEqualTo(iservFraDato);
+        assertThat(iservMapper.getIservSiden().truncatedTo(MILLIS)).isEqualTo(iservFraDato.truncatedTo(MILLIS));
```

Comment:
Assertj har også en 'isCloseTo()' assertion som er noe mer fleksibel. For eksempel, hvis du vil teste at tiden ligger innenfor 100 NANOS:
assertThat(iservMapper.getIservSiden()).isCloseTo(iservFraDato, new TemporalUnitWithinOffset(100, NANOS));

Men trunkering funker også fint.